identity_base_config/recipes/default.rb

```diff
@@ -64,11 +64,16 @@
   mode '0755'
 end
 
-cookbook_file '/usr/local/bin/id-git' do
-  source 'id-git'
-  owner 'root'
-  group 'root'
-  mode '0755'
+# download Slack hook/channel files for id-rails-console
+aws_account_id = AwsMetadata.get_aws_account_id
+
+s3_secret_url = "s3://login-gov.secrets.#{aws_account_id}-us-west-2/#{node.chef_environment}/"
+
+%w(slackwebhook slackchannel).each do |f|
+  execute "download #{f} from s3" do
```

Comment:
Whoa, these should be `file` resources, not `execute`.

https://github.com/18F/identity-devops/blob/9928e1ec250dfe55b9207686b1eca50bb01046e5/kitchen/cookbooks/config_loader/recipes/default.rb#L8-L10

Reply:
I was basing this off the behavior we currently have in `identity-elk`, e.g.:

https://github.com/18F/identity-devops/blob/9928e1ec250dfe55b9207686b1eca50bb01046e5/kitchen/cookbooks/identity-elk/recipes/es.rb#L162-L174